src/pages/Autolending/AutolendingStatus.vue

```diff
@@ -37,17 +37,10 @@
 
 				<kv-lightbox
 					class="autolending-status-lightbox"
-					:no-padding-bottom="true"
-					:no-padding-top="true"
 					:visible="showLightbox"
+					title="Change your auto-lending status"
 					@lightbox-closed="showLightbox = false"
 				>
-					<template slot="title">
-						<h3>
```

Comment:
The autolend h3 titles have different styling than the styling of the lightbox titles provided in this PR. Take a look at the dev env for reference.